### PR TITLE
Print fix

### PR DESCRIPTION
**Pull request by eino-makitalo**
Some 2to3 partial changes to be done for future python 3 work.
- exceptions
- print statements
- dictionary handlers
  These seems to go ok without extra modifications

### CODE REVIEW

**Comment by aaltat** [comment]
I am on holiday, so will take a look after the holidays are over.

**Comment by eino-makitalo** [comment]
Nice holiday. This is just small part of #479 (python 3)

**Comment by HelioGuilherme66** [comment]
Hi,
I merged your PR but it make the tests fail in Python 3.
Compare with my branch at [code_futurized with your PR merged](https://github.com/HelioGuilherme66/robotframework-selenium2library/tree/code_futurized_eino_print_fix)

Let me call to attention that we have several PRs for Python 3 compatibility, mine is #564.

**Comment by eino-makitalo** [comment]
Yes.  I check your combined PR and this change is small bite.   There are -  I agree - plenty of issues to check and fix to make same code base running both python2 / python 3.  I just tested in the first place those changes not violating python 2 code base and make code closer to python 3.  Python 3 working tests need also more work to get work back to  agile path.  I also run those tests with Windows / Python 3.5 and they failed but you cant eat whole cake.   So I just run those 2to3.py tests which did not break python 2 codebase.

**Comment by pekkaklarck** [comment]
Thanks for the PR. I've promised to take care of S2L Py3 support and will review related PRs as soon as I got time. Current main blockers are finishing our move to a new home and getting new S2L release (w/p Py3 support) out with @aaltat. Realistically that means I can concentrate on this task at the end of this month.

**Comment by aaltat** [comment]
The 1.8.0 is out and during the release process we changed few things. The changes caused a conflict(s) and by looking the pull request I can see at leas one problem. The `doc/buildhtml.py` was removed in 652410d2ca4adcd51f27f3fc97ff3524c522e726 Could you fix the problem? Also for my eyes it looks OK, but I am more of a Python 2 person than Python 3.

File: src/Selenium2Library/keywords/keywordgroup.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def _run_on_failure_decorator(method, *args, **kwargs):
 class KeywordGroupMetaClass(type):
     def __new__(cls, clsname, bases, dict):
         if decorator:
-            for name, method in dict.items():
+            for name, method in list(dict.items()):
```
Comment (pekkaklarck) [inline src/Selenium2Library/keywords/keywordgroup.py:31]: `list` isn't needed here.

File: src/Selenium2Library/locators/elementfinder.py [hunk 2]
```diff
@@ -22,7 +22,7 @@ def __init__(self):
             'default': self._find_by_default
         }
         self._strategies = NormalizedDict(initial=strategies, caseless=True, spaceless=True)
-        self._default_strategies = strategies.keys()
+        self._default_strategies = list(strategies.keys())
```
Comment (pekkaklarck) [inline src/Selenium2Library/locators/elementfinder.py:25]: `.keys()` isn't needed here.

File: test/lib/mockito/invocation.py [hunk 3]
```diff
@@ -47,14 +47,14 @@ def matches(self, invocation):
       return False
     if len(self.named_params) != len(invocation.named_params):
       return False
-    if self.named_params.keys() != invocation.named_params.keys():
+    if list(self.named_params.keys()) != list(invocation.named_params.keys()):
```
Comment (pekkaklarck) [inline test/lib/mockito/invocation.py:50]: `.keys()` isn't needed here.

File: test/resources/statuschecker.py [hunk 4]
```diff
@@ -177,14 +177,14 @@ def _get_log_message(self, msg_str):
     import os
 
     if not 2 <= len(sys.argv) <= 3 or '--help' in sys.argv:
-        print __doc__
+        print(__doc__)
```
Comment (pekkaklarck) [inline test/resources/statuschecker.py:180]: The whole `statuschecker.py` should actually be removed in favor of https://github.com/robotframework/statuschecker.

**Pull request merged**